recipes-bsp/u-boot/files/0017-spl-fit-Report-fdt-error-for-loading-u-boot.patch

```diff
@@ -17,7 +17,7 @@ Signed-off-by: Baocheng Su <baocheng.su@siemens.com>
  1 file changed, 6 insertions(+), 2 deletions(-)
 
 diff --git a/common/spl/spl_fit.c b/common/spl/spl_fit.c
-index a35be52965..00404935cb 100644
+index a35be529656..00404935cb2 100644
```

Comment:
I think maybe it's better to drop these meaningful changes.

Reply:
Means we need to pin to `--abbrev=10` - or switch to `--full-index` so that we no longer have to agree on the length.

Reply:
I am ok to both options, as long as we have an unified command set to generate the patch set, to make sure people do not step on each other.

I think we can put the command in somewhere for later reference, what about the wiki page, or the README.md?